bool is_reconize
geometry_msgs/Pose2D pose
int16 id
int16[] shape
int64[] crops